src/gobprepare/utils/postgres.py

```diff
@@ -39,3 +39,44 @@ def create_table_columnar_query(postgres_datastore: PostgresDatastore, tablename
     if _is_columnar_supported(postgres_datastore):
         return f"CREATE TABLE {tablename} ({columndefs}) USING columnar"
     return f"CREATE TABLE {tablename} ({columndefs})"
+
+def check_table_existence_query(schema: str, table_name: str) -> str:
+    """Return table exists query"""
+    return f"""SELECT EXISTS (
+                SELECT FROM pg_tables
+                WHERE schemaname = '{schema}'
+                AND tablename  = '{table_name}'
+            )"""
+
+def create_update_table_query(table_name: str, columns_values: list[dict[str]], conditions: list[dict[str]]) -> str:
```

Comment:
Also too complex. We only want to "update sync schema table for prepare end" and "update sync schema table for prepare start". That's much easier to understand than calling a generic query that introduces a lot of complexity because of it. Just create simple function with a simple action and write out the simple query so it's easy to see right away what's being done and how.

Reply:
De hulp functies zijn verwijderd en simple queries van gemaakt.
Ik heb deze generieke functies gebouwd zou dat ze (eventueel) gebruikt kunnen worden in andere implementaties.